webpack.config.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 module.exports = {
-  entry: './src/client/index.js',
+  entry: ['babel-polyfill', './src/client/index.js'],
```

Comment:
TODO: verify `heroku-postbuild` still works after merging in `babel-polyfill` ... see https://github.com/LearnersGuild/talent/pull/77/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R17